pokemon_pong.py

```diff
@@ -0,0 +1,168 @@
+'''Pokemon Pong by Alyssa Wu & Sandy Ta'''
+import pygame
+import sys
+import random
+import time
+from pygame.locals import *
+
+window_width = 1200
+window_height = 1200
+p1_wins = 0
+p2_wins = 0
+
+class Pikachu(pygame.sprite.Sprite): #making the Pikachu act as a Paddle
+
+    def __init__(self):
+        pygame.sprite.Sprite.__init__(self)
+        self.image = pygame.image.load('pikachu.png').convert_alpha()
+        self.image = pygame.transform.flip(self.image, 1, 0)
+        self.image = pygame.transform.scale(self.image, (150,200)) #resizing Pikachu
+        self.rect = self.image.get_rect() #converting the image into a rectangle for rect attributes
+        self.pikac = self.rect
+        self.pikac.centery = window_height / 2.0 
+        
+    def move_pika(self,key):
+        if 150 < self.rect.bottom < window_height: #parameters for where pikachu can move
+            if key[pygame.K_x]:
+                self.rect.centery += 25 #moves pikachu down 
+            elif key[pygame.K_s]:
+                self.rect.centery -= 25 #moves pikachu up
+
+        elif self.rect.bottom >= window_height: #if pikachu hits the bottom 
+            self.rect.centery -= 25
+        elif self.rect.bottom <= 150: #if pikachu hits the top 
+            self.rect.centery += 25
+
+class PongBall(pygame.sprite.Sprite):
+    def __init__(self, pika_one, pika_two):
+        pygame.sprite.Sprite.__init__(self)
+        pika = Pikachu()
+        pika2 = Pikachu2()
+        self.pika = pika_one
+        self.pika2 = pika_two
+        self.image = pygame.image.load('pokeball.png').convert_alpha()
+        self.image = pygame.transform.scale(self.image, (100,80)) 
+        self.rect = self.image.get_rect() 
+        self.ball = self.rect #making ball a rectangle to get rect attributes
+        self.ball.centery = window_height / 2.0 #ball starts at center of screen
+        self.ball.centerx = window_width / 2.0
+        self.dx = -22 #setting ball speed
+        self.dy = 5 #setting ball direction
+
+    def set_ball(self): #ball action at beginning of game
+        if self.ball.left > 0 or self.ball.right < window_width:
+            self.ball.move_ip(self.dx, self.dy)
+
+    def collision_wall(self):
+         #instances if ball hits a wall
```

Comment:
From a gameplay standpoint, it might be nice to either briefly pause, or wait for user response, to continue the game (so that after losing a point, the next one doesn't immediately fire off).